app/views/submissions/_publishing.html.haml

```diff
@@ -0,0 +1,6 @@
+- if submission.publishable? && submission.revocable?
+  = confirmable_action("Revoke publication", submission,
+      url: revoke_submission_path(submission), method: :patch)
+- elsif !submission.publishable?
```

Comment:
What can I say - I though this was agreed upon between you and @nowakowski - but perhaps I've got that wrong. Do you think we should change Submission.publishable to Submission.published?